app/views/quizzes/index.html.erb

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
There's a blank line here. You might want to think about adding [erb lint](https://github.com/Shopify/erb_lint) to your linter as it'll catch stuff like this. It'll also autofix the for you.